mpi-proxy-split/lower-half/mem-wrapper.cpp

```diff
@@ -47,35 +47,28 @@ using namespace std;
 #define _real_mmap mmap
 #define _real_munmap munmap
 
+#define UH_BASE_ADDR ((void *)0xC0000000) // 3GB
```

Comment:
As discussed together, maybe this should first look for an environment variable, like `MANA_UH_BASE_ADDR`.  Then it's easy for a user to change this.  But I'll approve this in advance.